tutorials/forest-vegetation-condition/index.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Monitoring forest vegetation condition
+description: Estimate forest vegetation status over the years and infer condition using linear trend analysis
+author: pskoulgi
+tags: forest, vegetation, trend-analysis
+date_published: 2020-06-12
+---
+<!--
+Copyright 2019 The Google Earth Engine Community Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+[Open & run code in Earth Engine code editor](https://code.earthengine.google.co.in/2ac326e0fc824134f1dd1c7575591151)
```

Comment:
Please remove code editor link, since we can't review code in code editor.

Reply:
Done